optimum/habana/diffusers/pipelines/pipeline_utils.py

```diff
@@ -344,6 +344,16 @@ def is_saveable_module(name, value):
                 create_pr=create_pr,
             )
 
+    def to(self, *args, **kwargs):
+        """
+        Intercept to() method and disable gpu-hpu migration before sending to diffusers
+        """
+        kwargs["hpu_migration"] = False
```

Comment:
The arg `hpu_migration` is defined at SynapseAI's level right?

Reply:
@regisss, no we define it as a "guardrail" not to apply migration when diffusers pipeline assignment comes from OH, if we manage to push this commit in HF diffusers:  https://github.com/huggingface/diffusers/compare/main...dsocek:diffusers:hpu_support

Basically the idea is to enable pure diffusers lib to run any pipeline on Gaudi out-of-the-box simply by moving the pipeline `.to("hpu")`.  But with optimized diffusers pipelines which come from OH, we don't need this migration to kick in.

This enablement requires 2 step merge process:
1. set guardrail - this PR in OH
2. enable generic diffusers pipelines in HF diffusers - future PR based on  https://github.com/huggingface/diffusers/compare/main...dsocek:diffusers:hpu_support draft

Reply:
Sounds good!